AutomatedTesting/Gem/PythonTests/Terrain/EditorScripts/Terrain_World_ConfigurationWorks.py

```diff
@@ -0,0 +1,163 @@
+    import editor_python_test_tools.hydra_editor_utils as hydra
+    import azlmbr.math as azmath
+    import azlmbr.legacy.general as general
+    import azlmbr.bus as bus
+    import azlmbr.editor as editor
+    import azlmbr.terrain as terrain
+    import math
+
+    SET_BOX_X_SIZE = 2048.0
+    SET_BOX_Y_SIZE = 2048.0
+    SET_BOX_Z_SIZE = 100.0
+    CLAMP = 1
+
+    helper.init_idle()
+    
+    # 1) Load the level
+    helper.open_level("", "Base")
+    helper.wait_for_condition(lambda: general.get_current_level_name() == "Base", 2.0)
+
+    # 2) Load the level components
+    terrain_world_component = hydra.add_level_component("Terrain World")
+    hydra.add_level_component("Terrain World Renderer")
+
+    # 3) Create 2 test entities, one parent at 512.0, 512.0, 50.0 and one child at the default position and add the required components
+    entity1_components_to_add = ["Axis Aligned Box Shape", "Terrain Layer Spawner", "Terrain Height Gradient List", "Terrain Physics Heightfield Collider"]
+    entity2_components_to_add = ["Vegetation Reference Shape", "Gradient Transform Modifier", "FastNoise Gradient"]
+    terrain_spawner_entity = hydra.Entity("TerrainEntity")
+    terrain_spawner_entity.create_entity(azmath.Vector3(512.0, 512.0, 50.0), entity1_components_to_add)
+    Report.result(Tests.create_terrain_spawner_entity, terrain_spawner_entity.id.IsValid())
+    height_provider_entity = hydra.Entity("HeightProviderEntity")
+    height_provider_entity.create_entity(azmath.Vector3(0.0, 0.0, 0.0), entity2_components_to_add,terrain_spawner_entity.id)
+    Report.result(Tests.create_height_provider_entity, height_provider_entity.id.IsValid())
+
+    # Give everything a chance to finish initializing.
+    general.idle_wait_frames(1)
+
+    # 4) Start the Tracer to catch any errors and warnings
+    with Tracer() as section_tracer:
+        # 5) Set the base Terrain World values
+        world_bounds_max = azmath.Vector3(1024.0, 1024.0, 1024.0)
+        world_bounds_min = azmath.Vector3(0.0, 0.0, 0.0)
+        height_query_resolution = azmath.Vector2(1.0, 1.0)
+        editor.EditorComponentAPIBus(bus.Broadcast, "SetComponentProperty", terrain_world_component, "Configuration|World Bounds (Max)", world_bounds_max)
+        editor.EditorComponentAPIBus(bus.Broadcast, "SetComponentProperty", terrain_world_component, "Configuration|World Bounds (Min)", world_bounds_min)
+        editor.EditorComponentAPIBus(bus.Broadcast, "SetComponentProperty", terrain_world_component, "Configuration|Height Query Resolution (m)", height_query_resolution)
+        world_max = hydra.get_component_property_value(terrain_world_component, "Configuration|World Bounds (Max)")
+        world_min = hydra.get_component_property_value(terrain_world_component, "Configuration|World Bounds (Min)")
+        world_query = hydra.get_component_property_value(terrain_world_component, "Configuration|Height Query Resolution (m)")
```

Comment:
consider adding hydra.set_component_property_value() to the helper or switch to using editor_entity_utils.py instead which provides EditorComponent class that has set_component_property_value() using that helper it would be
terrain_world_component.set_component_property_value("Configuration|World Bounds (Max)", world_bounds_max)

Reply:
Added function to hydra and changed code.